experimental/schemabuilder/reflector.go

```diff
@@ -146,6 +146,10 @@ func NewSchemaBuilder(opts BuilderOptions) (*Builder, error) {
 	}, nil
 }
 
+func (b *Builder) Reflector() *jsonschema.Reflector {
```

Comment:
this is helpful to let downstream tests know what the reflector found and how it can control output